client/rpc-servers/src/lib.rs

```diff
@@ -104,7 +105,7 @@ pub fn start_ws<M: Send + Sync + 'static>(
 	let mut builder = WsServerBuilder::new()
 		.max_request_body_size(max_request_body_size as u32)
 		.max_connections(max_connections as u64)
-		.set_id_provider(RandomStringIdProvider::new(16))
+		//.set_id_provider(id_provider.unwrap())
```

Comment:
we need to change to API in jsonrpsee for this work.

//cc @maciejhirsz @dvdplm WDYT? Otherwise, I think many will get angry at us by enforcing `type param` on the `Configuration` :)